omod/src/main/java/org/openmrs/module/fhir/resources/FHIRRelatedPersonResource.java

```diff
@@ -1,17 +1,32 @@
 package org.openmrs.module.fhir.resources;
 
+import ca.uhn.fhir.rest.param.ReferenceParam;
+import ca.uhn.fhir.rest.param.TokenParam;
 import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.dstu3.model.Patient;
 import org.hl7.fhir.dstu3.model.RelatedPerson;
 import org.openmrs.api.context.Context;
 import org.openmrs.module.fhir.api.RelatedPersonService;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class FHIRRelatedPersonResource extends Resource {
 
 	public RelatedPerson getByUniqueId(IdType id) {
 		RelatedPersonService relatedPersonService = Context.getService(RelatedPersonService.class);
 		return relatedPersonService.getRelatedPerson(id.getIdPart());
 	}
 
+	public List<RelatedPerson> searchByPatient(ReferenceParam identifier) {
+		List<RelatedPerson> relatedPeople = new ArrayList<>();
+		if (RelatedPerson.SP_PATIENT.equals(identifier.getChain())) {
```

Comment:
Technically the value you match here should be `RelatedPerson:patient` and not just `patient`. I just realized we have some outdated examples on the HAPI FHIR website here.. `_include=patient` was the syntax in DSTU2 but it was changed in FHIR DSTU3+.

Reply:
will make the change